src/types/network.ts

```diff
@@ -5,3 +5,7 @@ export const Network = {
   mainnet: 'mainnet',
   testnet: 'testnet',
 } as const
+
+export const GlobalNetwork = 'global'
```

Comment:
I'm confused

Reply:
The idea is that we support two kinds of search. One is launched from within a page belonging to a specific network. The URL is like `/mainnet/ember/search?q=whatever`. In this case, we are mainly showing the results from that network. (And maybe some indicators about results elsewhere.)

Then there is the so-called "global" search, which is what you get if you launch a search right from the opening homepage, without actually entering any paratime first. The URL for this is `/global/search?q=whatever`. In this case, you see all results, grouped by networks of course.

So, in the URL, there is something that specifies the flavor of the search. That is either a name of a network (mainnet or testnet), or "global" for global search.

Hence, we have the `NetworkOrGlobal` type, for these values.

Reply:
@donouwens do we ever need a global search?

Reply:
We discussed this (somewhat) on slack, and we decided that we probably do. We can discuss this further at the upcoming team sync. But for now, I think it works...